charts/nx-cloud/templates/_images.yml

```diff
@@ -61,3 +61,10 @@ Return proper aggregator image name
 {{- define "nxCloud.images.aggregator.image" }}
 {{- include "nxCloud.images.common" (dict "imageRoot" .Values.aggregator.image "global" .Values.global "image" .Values.image) }}
 {{- end }}
+
+{{/*
+Return proper messagequeue image name
+*/}}
+{{- define "nxCloud.images.messagequeue.image" }}
+{{- include "nxCloud.images.common" (dict "imageRoot" .Values.messagequeue.image "global" .Values.global "image" .Values.image) }}
```

Comment:
I have a question....because I don't feel like I understand Helm syntax well enough yet 😄 

what the above does is it essentially allows to set different tags for the different services - for example, if we want to hotfix only the prod API?

but for most installations, enterprise clients will just set the the image tag via the `helm upgrade .... --set global.imageTag=2305.17.1` right, and it will use the same tag for everything?